shared/oval/installed_env_is_a_container.xml

```diff
@@ -0,0 +1,19 @@
+<def-group>
+  <definition class="compliance" id="is.container" version="2">
+    <metadata>
+      <title>Check if the scan target is a container</title>
+      <description>Check if file /.dockerenv exists, if it does then we consider to be a docker filesystem.</description>
+    </metadata>
+    <criteria>
+      <criterion comment="Check if /.dockerenv exists" test_ref="is.container" />
+    </criteria>
+  </definition>
+
+  <unix:file_test check="all" check_existence="all_exist" comment="Check if /.dockerenv exists" id="is.container" version="1">
+    <unix:object object_ref="is.container" />
+  </unix:file_test>
+
+  <unix:file_object comment="Check file /.dockerenv" id="is.container" version="1">
```

Comment:
I would keep is.container and add more definitions when we need to add support for more tech.